<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PVZ_Cheater</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <widget class="QLabel" name="lbGameIsRunningColor">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	background-color:rgb(255, 0, 0);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbGameIsRunning">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;宋体&quot;;</string>
         </property>
         <property name="text">
          <string>检测中...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>阳光</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>阳光数值：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leSunCount">
             <property name="toolTip">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbChangeSunCount">
             <property name="text">
              <string>确定</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbLockSunCount">
          <property name="text">
           <string>锁定阳光数值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>自动收集阳光</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QRadioButton" name="rbNormalCollectSun">
             <property name="text">
              <string>不自动收集</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbQuickAutoCollectSun">
             <property name="text">
              <string>快速自动收集</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbSlowAutoCollectSun">
             <property name="text">
              <string>较慢速自动收集</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbPlantSunCount">
          <property name="title">
           <string>种植物</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QRadioButton" name="rbPlantNormalSunCount">
             <property name="text">
              <string>正常</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbPlantAddSunCount">
             <property name="text">
              <string>种植物加阳光</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbPlantDoNotSubSunCount">
             <property name="text">
              <string>种植物不减阳光</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbCollectSunCount">
          <property name="title">
           <string>收集阳光</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QRadioButton" name="rbCollectNormalSunCount">
             <property name="text">
              <string>正常</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbCollectSubSunCount">
             <property name="statusTip">
              <string>收集后减阳光(可能会减为负数,但游戏界面仍显示0)</string>
             </property>
             <property name="text">
              <string>收集后减阳光(可能会减为负数,但仍显示0)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbCollectDoNotAddSunCount">
             <property name="text">
              <string>收集后不加阳光</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>卡槽</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="gbPlantSunCount_2">
          <property name="title">
           <string>冷却时间</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label">
             <property name="statusTip">
              <string>卡槽CD每次更新差值,值越大冷却越快,默认为1,为0时暂停冷却</string>
             </property>
             <property name="text">
              <string>每次更新冷却进度差值Δ[-128,127](值越大冷却越快,默认为1)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QSpinBox" name="sbCardCDDelta">
                <property name="statusTip">
                 <string>[-128,127] 默认为1,为0暂停冷却,为负数冷却进度后退到-2^31可能崩溃</string>
                </property>
                <property name="minimum">
                 <number>-128</number>
                </property>
                <property name="maximum">
                 <number>127</number>
                </property>
                <property name="value">
                 <number>1</number>
                </property>
                <property name="displayIntegerBase">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbCardCDDelta">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>确定</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbCardCDDeltaDefault">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>默认</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="default">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>415</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
